modules/ROOT/pages/managing-business-groups.adoc

```diff
@@ -16,17 +19,150 @@ What you can view and edit depends on your permissions.
 include::partial$include-nav-steps-org-admin.adoc[]
 
 . In the Access Management navigation menu, click *Business Groups*.
-. Click the name of the root organization.
+. Click the name of the root business group.
+
+[[create-business-group]]
+== Create a Business Group
+
+When you create a new business group under the top-level (root), business group, all the current users with the Organization Administrator permission in the root business group appear in the list of users in the newly created business group.
+
+include::partial$include-nav-steps-org-admin.adoc[]
+
+include::partial$include-nav-business-groups.adoc[]
+
+. To create the root business group for your organization, click *Create business group*.
+. To create a child business group, click the *...* menu next to the business group that will be the parent.
+. Click *Create child group*.
+. In the dialog box, enter:
+  * *Business Group name*: Name for the new business group.
```

Comment:
Consider making these descriptions parallel (either all noun or imperative). And adjust punctuation, as needed.

Reply:
I'll address this when I go back in later to verify all the procedures in this topic.